packages/convertim/src/Model/Order/ConvertimOrderDataToOrderDataMapper.php

```diff
@@ -175,9 +186,25 @@ protected function mapConvertimTransportDataToOrderItem(
         OrderData $orderData,
     ): void {
         $transport = $this->transportFacade->getByUuid($convertimOrderTransportData->getUuid());
+
         $transportOrderItemData = $this->orderItemDataFactory->create(OrderItemTypeEnum::TYPE_TRANSPORT);
         $transportOrderItemData->transport = $transport;
         $transportOrderItemData->name = $transport->getName();
+
+        $pickupPlaceIdentifier = $convertimOrderTransportData->getExtra()?->getPickUpPointCode();
+
+        if ($transport->isPersonalPickup()) {
+            try {
+                $store = $this->storeFacade->getByUuidAndDomainId(
+                    $pickupPlaceIdentifier,
+                    $this->domain->getId(),
+                );
+
+                $transportOrderItemData->name .= ' ' . $store->getName();
```

Comment:
Could you please somehow extract this logic so it is not duplicated with the code in `PersonalPickupPointMiddleware`? :pray:

Reply:
I have tried this and we have discussed it and it brought more problems than solved.